bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -2,8 +2,8 @@
 module_id: bash_103_combining_commands
```

Comment:
This comment is actually for line 298 of this file (I can't put it there, THANKS GITHUB).  In the sentence 

> The vertical line | is called a pipe. On American qwerty keyboards symbol | is located on the same key as the backslash \ and can be typed by using the shift key with \ .

 I think you should add a "the" before symbol, like this:

> The vertical line | is called a pipe. On American qwerty keyboards the symbol | is located on the same key as the backslash \ and can be typed by using the shift key with \ .

Reply:
fixed!